core/quantity/src/Formatter/Format.ts

```diff
@@ -326,6 +359,37 @@ export class Format extends BaseFormat {
       if (undefined === this.units || this.units.length === 0)
         throw new QuantityError(QuantityStatus.InvalidJson, `The Format ${this.name} has a Composite with no valid 'units'`);
     }
+
+    if(this.type === FormatType.Azimuth || this.type === FormatType.Bearing) {
+      // these units cannot be loaded from loadFormatProperties() because they require an async call, and the method signature is already public
```

Comment:
loadFormatProperties is public but it is in a beta class

Reply:
That is true, I did consider making it async, but there is one call from ECFormat.fromJSONSync()
... And now I realize, that fromJSONSync in metadata has a copy of what this method here does, so we'll have to put a duplicate of this code there.... meh. I'll bring this up with Anmol and Nam.

Reply:
I've added dedupe of method calls to the backlog as part of quantity formatting.